README.md

```diff
@@ -504,10 +504,10 @@ To view sibling stats:
 
 ```lua
 > siblings()
-Address                             Sucesses  Failures     Note
-192.168.1.79:4001                   18        7
-192.168.1.30:4001                   25        0
-192.168.1.54:4001                   0         0            Self
+Address                             Send Successes  Send Failures  Rcv Successes   Rcv Failures     Note
+192.168.1.79:4001                   18              7              0               0
+192.168.1.30:4001                   25              0              0               0
+192.168.1.54:4001                   0               0              0               0                Self
 ```
```

Comment:
The old output hasn't been correct since something.

I'm not entirely certain about the new output, but it's closer (I copied the style from elsewhere in this repository and it seems to more closely match the code that should be generated it).

Reply:
I need to cleanup the README.md examples anyway so don't worry too much about the output